enterprise/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -17,28 +17,51 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: enterprise/admin/forms.py:21
+#: enterprise/admin/actions.py:68
+msgid "Unlink selected objects from existing course catalogs"
+msgstr ""
+
+#: enterprise/admin/forms.py:22
 msgid "Email or Username"
 msgstr ""
 
-#: enterprise/admin/forms.py:40
-#, python-brace-format
+#: enterprise/admin/forms.py:41
 msgid ""
 "{email_or_username} does not appear to be a valid email or known username"
 msgstr ""
 
-#: enterprise/admin/forms.py:47
-#, python-brace-format
+#: enterprise/admin/forms.py:48
 msgid ""
 "User with email {email} is already registered with Enterprise Customer "
 "{ec_name}"
 msgstr ""
 
+#: enterprise/admin/forms.py:86
+msgid "None"
+msgstr ""
+
 #: enterprise/admin/views.py:129
-#, python-brace-format
 msgid "Email {email} is not linked to Enterprise Customer {ec_name}"
 msgstr ""
 
+#: enterprise/course_catalog_api.py:45
```

Comment:
Is it good to keep the line number here ? I meant, any upcoming change in code will not change the line number for django translation ?